app/controllers/team_controller.rb

```diff
@@ -0,0 +1,6 @@
+class TeamController < ApplicationController
+  def index
```

Comment:
Is better to use [scope](http://guides.rubyonrails.org/active_record_querying.html#scopes), it let your test more easy.

```
      @active_developers = Admin::Developer.actives
      @inavtive_developers = Admin::Developer.inactives
```

This istructions is unnecessary: 
```order(active: :desc)```
or explaning it to me.